orange/src/test/java/com/hartwig/hmftools/orange/algo/linx/DnaFusionSelectorTest.java

```diff
@@ -97,6 +98,30 @@ public void canSelectFusionOfOncogene()
         assertNotNull(findByName(fusions, "with three onco"));
     }
 
+    @Test
+    public void canSelectAdditionalViableFusions()
+    {
+        LinxFusion inFrameFusion = LinxTestFactory.fusionBuilder()
+                .phased(FusionPhasedType.INFRAME)
+                .chainTerminated(false)
+                .name("in frame fusion")
+                .build();
+        LinxFusion outOfFrameFusion = LinxTestFactory.fusionBuilder()
+                .phased(FusionPhasedType.OUT_OF_FRAME)
+                .chainTerminated(false)
+                .name("out of frame fusion")
+                .build();
```

Comment:
Fairly minor but your test relies on `fusionBuilder` to have reported=false as default. It may make your test more readable and resilient to explicitly set this parameter. Also that would be consistent with the other tests in this class.